x-pack/plugins/enterprise_search/public/applications/app_search/components/log_retention/components/log_retention_callout.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useValues } from 'kea';
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+
+import { EuiLinkTo } from '../../../../shared/react_router_helpers';
+
+import { AppLogic } from '../../../app_logic';
+import { SETTINGS_PATH } from '../../../routes';
+import { ANALYTICS_TITLE } from '../../analytics';
+import { API_LOGS_TITLE } from '../../api_logs';
+
+import { LogRetentionLogic, LogRetentionOptions, renderLogRetentionDate } from '../';
+
+const TITLE_MAP = {
+  [LogRetentionOptions.Analytics]: ANALYTICS_TITLE,
+  [LogRetentionOptions.API]: API_LOGS_TITLE,
+};
+
+interface Props {
+  type: LogRetentionOptions;
+}
+export const LogRetentionCallout: React.FC<Props> = ({ type }) => {
+  const { logRetention } = useValues(LogRetentionLogic);
+  const {
+    myRole: { canManageLogSettings },
+  } = useValues(AppLogic);
+
+  const logRetentionSettings = logRetention?.[type];
+  const title = TITLE_MAP[type];
+
+  const hasILM = logRetention !== null;
```

Comment:
I know [ILM is an Elastic acronym](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-lifecycle-management.html), but its weird that we refer to this as "Log Retention" everywhere except for a few boolean values.  Worth dropping the acronym internally? `hasLogRetention` `hasLogRetentionDisabled`?

Reply:
I have no strong preference, I think my only hesitation is there are some instances where ILM does actually reference to separate Elasticsearch concept/architecture that can be disabled separately, whereas Enterprise Search can override ILM (mostly by disabling it), at which point there's the broader concept of "Log Retention" separate from ILM.

I think I'd slightly prefer to leave it as-is unless we can check with Chris/back-end engineers to make sure our understanding is correct?

Reply:
I ended up liking this way more over time and updated the tooltip & callout components to use your suggestion in a448f12 - thanks Byron!!